dashboard/lib/pd/foorm/survey_reporter.rb

```diff
@@ -116,5 +154,19 @@ def self.get_raw_data_for_workshop(workshop_id)
 
       [ws_submissions, foorm_submissions, forms]
     end
+
+    # @param integer workshop_id: id for a workshop
+    # @return {facilitator_id: facilitator_name,...} object with data
+    # for each facilitator for the workshop specified
+    def self.get_facilitators_for_workshop(workshop_id)
```

Comment:
Seems like could go on workshop model? I guess currently specific to Foorm but might be useful otherwise in the future.

Reply:
I should probably rename to be something like get_formatted_facilitators... because the function is mostly taking the facilitators list from the workshop and pulling out the relevant info we want (id and name), which is fairly specific to the pipeline